var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -220,6 +220,13 @@ def url_for_version(self, version):
         url = "http://www.open-mpi.org/software/ompi/v{0}/downloads/openmpi-{1}.tar.bz2"
         return url.format(version.up_to(2), version)
 
+    @property
+    def headers(self):
+        hdrs = HeaderList(find(self.prefix.include, 'mpi.h', recursive=False))
+        if not hdrs:
+            hdrs = HeaderList(find(self.prefix, 'mpi.h', recursive=True))
+        return hdrs or None
```

Comment:
given discussion in another PR, we may want to thrown an error instead

Reply:
@v-dobrev please do that!

Reply:
If my PR #7277 is approved, then returning `None` will throw an error. If it is declined, then I will change this PR. So, let's wait for the decision on 7277.

Reply:
Ok #7277 is now merged!